plugins/modules/ocapi_command.py

```diff
@@ -0,0 +1,268 @@
+      category: Chassis
+      command: PowerModeNormal
+      baseuri: "{{ baseuri }}"
+      username: "{{ username }}"
+      password: "{{ password }}"
+  - name: Set chassis indicator LED to on
+    community.general.ocapi_command:
+      category: Chassis
+      command: IndicatorLedOn
+      baseuri: "{{ baseuri }}"
+      proxy_slot_number: 2
+      username: "{{ username }}"
+      password: "{{ password }}"
+  - name: Set chassis indicator LED to off
+    community.general.ocapi_command:
+      category: Chassis
+      command: IndicatorLedOff
+      baseuri: "{{ baseuri }}"
+      proxy_slot_number: 2
+      username: "{{ username }}"
+      password: "{{ password }}"
+  - name: Reset Enclosure
+    community.general.ocapi_command:
+      category: Systems
+      command: PowerGracefulRestart
+      baseuri: "{{ baseuri }}"
+      proxy_slot_number: 2
+      username: "{{ username }}"
+      password: "{{ password }}"
+  - name: Firmware Upload
+    community.general.ocapi_command:
+      category: Update
+      command: FWUpload
+      baseuri: "iom1.wdc.com"
+      proxy_slot_number: 2
+      username: "{{ username }}"
+      password: "{{ password }}"
+      update_image_path: "/path/to/firmware.tar.gz"
+  - name: Firmware Update
+    community.general.ocapi_command:
+      category: Update
+      command: FWUpdate
+      baseuri: "iom1.wdc.com"
+      proxy_slot_number: 2
+      username: "{{ username }}"
+      password: "{{ password }}"
+  - name: Firmware Activate
+    community.general.ocapi_command:
+      category: Update
+      command: FWActivate
+      baseuri: "iom1.wdc.com"
+      proxy_slot_number: 2
+      username: "{{ username }}"
+      password: "{{ password }}"
+  - name: Delete Job
+    community.general.ocapi_command:
+      category: Jobs
+      command: DeleteJob
+      job_name: FirmwareUpdate
+      baseuri: "{{ baseuri }}"
+      proxy_slot_number: 2
+      username: "{{ username }}"
+      password: "{{ password }}"
+'''
+
+RETURN = '''
+msg:
+    description: Message with action result or error description
+    returned: always
+    type: str
+    sample: "Action was successful"
+
+jobUri:
+    description: URI to use to monitor status of the operation.  Returned for async commands such as Firmware Update, Firmware Activate.
+    returned: when supported
+    type: str
+    sample: "https://ioma.wdc.com/Storage/Devices/openflex-data24-usalp03020qb0003/Jobs/FirmwareUpdate/"
+
+operationStatusId:
+    description: OCAPI State ID (see OCAPI documentation for possible values)
+    returned: when supported
+    type: int
+    sample: 2
+
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.general.plugins.module_utils.ocapi_utils import OcapiUtils
+from ansible.module_utils.common.text.converters import to_native
+try:
+    from urlparse import urljoin  # Python 2
+except ImportError:
+    from urllib.parse import quote_plus, urljoin  # Python 3+
```

Comment:
Please use six (vendored in ansible-core) to import the correct symbols.

Reply:
Done